### DISCUSSION TITLE
Change arrow on Popover

### DISCUSSION THREAD

**Discussion by theo-mesnil** [ariakit/ariakit#775] [Questions] [answered]
Hi, i would like to know if it's possible to change the arrow of the Popover component. We need to remove the rounded corner. Thanks :)

**Comment by diegohaz** [comment]
@theo-mesnil It is! You just need to replace `PopoverArrow`'s children. This is the default one: https://github.com/reakit/reakit/blob/3ac8395575e25848a8f23fd38bab9678e97ba9a7/packages/reakit/src/Popover/PopoverArrow.tsx#L48-L58

**Comment by theo-mesnil** [comment]
Ok thanks :) i've got a problem with the placement, i think the transform function do not work properly

<img width="1098" alt="Capture d’écran 2020-10-26 à 15 30 29" src="https://user-images.githubusercontent.com/50322149/97185355-3bc67480-17a0-11eb-8493-e596e98a0a89.png">

```jsx
<S.Arrow {...rest} style={arrowStyle}>
   <svg height="30" width="30" xmlns="http://www.w3.org/2000/svg">
      <path d="M6 30l9-10 9 10z" fill="currentColor" fillRule="nonzero" />
   </svg>
</S.Arrow>
```

S.Arrow it's a styled component from PopoverArrow

Thanks !

**Comment by theo-mesnil** [comment, marked as answer]
For now i just do the same transform like in the PopoverArrow component
